### DISCUSSION TITLE
Scroll to zoom : an option ?

### DISCUSSION THREAD

**Discussion by Grrishka** [Zettlr/Zettlr#3288] [Q&A]
Hello,

I didn't understand why the scroll + Ctrl zoom feature was removed. I read that 
> _Scroll-to-zoom has been removed, because too many people have accidentally held down Cmd/Ctrl while using the scrollwheel, causing many accidental zoom operations._

What kind of accident can compromise the Ctrl (or Cmd, or Shift) + scroll wheel function that we find in all word processing, image or web browsers?

Can't we put it as an option to be disabled for the few people who don't know how to use their scroll wheel?

Am I alone in needing to regularly adjust the size of my work surface?

**Comment by nathanlesage** [comment]
Just for some context: I removed that feature again because I had a REALLY large crowd demanding it to be removed, and since (back at that time) no one really needed it, I removed it without any option. If enough people speak up here, I'll consider re-adding this (doesn't Ctrl +/Ctrl - do the job for you?)

**Reply by Grrishka** [reply]
Thanks for the answer ;)

Zettlr is designed to work only with the keyboard and that's fine! But I am very surprised that people have asked to remove this feature... but ok!

Doesn't Ctrl +/Ctrl - do the job for me ? No unfortunately! Like other users with Debian, "Ctrl -" doesn't work* (it doesn't work with my Manjaro either: Zettlr 2.2.3 on both my PCs).

And on my laptop, I like to adjust the view by sliding my two fingers on the mouse-pad.

Hey people! Ask massively for the return of the scroll to zoom ;D

Thanks again for this nice software.

_[Translated with www.DeepL.com/Translator (free version)]_

*Edit : I have a French keyboard, AZERTY

**Reply by kyaso** [reply]
I'd also welcome a return of the Ctrl+Scroll zoom feature :D

I don't see any obstacle of adding it as an option in the preferences?

**Reply by elekis-git** [reply]
+1;  

thanks
